android/app/buildConfigs.gradle

```diff
@@ -13,7 +24,7 @@ def createBuildConfig(buildConfigName) {
     logger.quiet("Using build config $buildConfigName")
 
     def command = "yarn --silent babel-node tools/build-config to-json $buildConfigName"
-    def proc = command.execute()
+    def proc = execCommand(command)
     proc.waitFor()
```

Comment:
If it works only waitFor, it should be removed here as well.

Reply:
will be done

Reply:
The waitFor is necessary in this case, otherwise it runs into a "not exited" error.

Reply:
See https://github.com/Integreat/integreat-react-native-app/pull/331/files#r492143073. The check for `proc.exitValue()` has to be after `proc.in.text`.

Reply:
At my tests changing this workflow results in an process error at CI.